### ISSUE TITLE
90% usage of GPU after upgrade to 2.11

### ISSUE THREAD

**Issue by michalterbert** [osy86/HaC-Mini#324]
**Describe the bug**
90% usage of GPU after upgrade to 2.11.
I have observed strange behave  after upgrade to 2.11
Quartz Debug.app shows load about 70-80% (after reboot, no application running)
iStat menus also showed almost 100% usage of RAM 
On chrome/safari I have observed glitches during scrolling. Full screen videos on youtube has dropped frames.
One good thing - no flickering 👍 

**Expected behavior**
Load should be 30-40% - after system reboot, without application.

**Configuration**
- HaC Mini version [e.g: 2.0]: 2.11
- OSX version [e.g: 10.14.6]: 10.15.6
- NUC model [e.g: NUC8i7HNK]: NUC8i7HVK
- Boot SSD model and protocol [e.g: Samsung EVO 860 1TB, SATA]: SSD ADATA XPG SX8200 PRO 1TB NVME
- Second SSD model and protocol (if installed) [e.g: HP EX920 1TB, NVMe]: no
- First DIMM model [e.g: Patriot DDR4-2666 16GB]: Kingston DDR4-2666 16GB
- Second DIMM model (if installed): Kingston DDR4-2666 16GB
- Display port(s) used [e.g: USB-C, mini DP]: mini DP
- Display resolution(s) [e.g: 4K, 5K]:  4K
- Any TB3 devices: 
- Any USB devices: Keyboard, mouse

**Modifications**
No modifications.

**Panic log**
Today  Im back to 2.10 so on below screens you can see when issue was.
<img width="646" alt="Zrzut ekranu 2020-08-29 o 09 42 45" src="https://user-images.githubusercontent.com/15417117/91631735-1dc8d780-e9dc-11ea-901b-aa27843a5f93.png">ay
<img width="282" alt="Zrzut ekranu 2020-08-29 o 09 42 49" src="https://user-images.githubusercontent.com/15417117/91631737-1e616e00-e9dc-11ea-98ab-2b5a0103e3c4.png">
<img width="646" alt="Zrzut ekranu 2020-08-29 o 09 42 37" src="https://user-images.githubusercontent.com/15417117/91631732-1acde700-e9dc-11ea-9a70-d6196cb80117.png">

Yes, now I have install 2.10 and load on GPU is ok - 30%-40% with chrome running. But screen flickering issue is also back :(
<img width="314" alt="Zrzut ekranu 2020-08-29 o 09 45 00" src="https://user-images.githubusercontent.com/15417117/91631761-4fda3980-e9dc-11ea-93a2-6694a077a2b8.png">

**Comment by osy86** [comment]
Seems like there’s been some graphics drivers issues in latest OSX update. Did you observe any issues before the update?

**Comment by michalterbert** [comment]
Nope, I have that problem only on 2.11.
Now Im on 2.10 and "load" issue gone, but sometimes I have screen flickering issue.

**Comment by cyuet** [comment]
> Nope, I have that problem only on 2.11.
> Now Im on 2.10 and "load" issue gone, but sometimes I have screen flickering issue.

yes, when my main monitor using 2K or 4K mode, it will have screen flickering issue, but 1080P is normal

**Comment by KrisCris** [comment]
Hmm, I think the VRAM usage issue is long-standing... Not sure if this is really happening or just iStat displays it incorrectly.

<img width="585" alt="截屏2020-10-02 下午11 22 40" src="https://user-images.githubusercontent.com/38860226/94940503-2b570d00-0506-11eb-9e87-337491f46e08.png">

**Comment by bluetooth29** [comment]
![截屏2020-11-27 上午2 55 42](https://user-images.githubusercontent.com/74524619/100384900-36d53900-305c-11eb-93bb-1644ccc3b25a.png)
